VideoLocker/src/main/java/org/edx/mobile/module/notification/EdxParsePushBroadcastReceiver.java

```diff
@@ -0,0 +1,38 @@
+package org.edx.mobile.module.notification;
+
+import com.parse.ParsePushBroadcastReceiver;
+
+import org.edx.mobile.view.CourseDetailTabActivity;
+import org.edx.mobile.view.MyCoursesListActivity;
+
+/**
+ * subclass ParsePushBroadcastReceiver to provide fine control of
+ * app's behavior on receiving the notification message
+ */
+public class EdxParsePushBroadcastReceiver extends ParsePushBroadcastReceiver {
```

Comment:
@aleffert because we also import Parse libraries. it has classes like Parse, ParsePush, ...